core/src/main/java/com/sequenceiq/cloudbreak/logger/LogContextAspects.java

```diff
@@ -20,6 +24,9 @@ public void interceptCloudbreakEventHandlerAcceptMethod() {
     @Before("com.sequenceiq.cloudbreak.logger.LogContextAspects.interceptCloudbreakEventHandlerAcceptMethod()")
     public void buildLogContextForReactorHandler(JoinPoint joinPoint) {
         com.sequenceiq.flow.logger.LogContextAspects.buildLogContextForEventHandler(joinPoint);
-        LOGGER.debug("A Reactor event handler's 'accept' method has been intercepted: {}, MDC logger context is built.", joinPoint.toShortString());
+        Event<CloudbreakCompositeEvent> event = (Event<CloudbreakCompositeEvent>) joinPoint.getArgs()[0];
```

Comment:
isn't it a bit risky to get the first item? can't this throw a nullpointer?
also, are you sure about the cast? shall we check it before?

Reply:
This method always has an `Event<CloudbreakCompositeEvent>` argument.